timezone_converter/list_view.py

```diff
@@ -9,12 +9,20 @@
 
 
 class ListView(Helper):
+    def __init__(self, letters: List[str]) -> None:
```

Comment:
As we will use `[None]` if no value is supplied, we need to change the type signature of `letters` to: `List[Optional[str]]` (and also import `Optional` as we were not using it before)

Reply:
AS we will use `list(string.ascii_lowercase)` there is no need to change this anymore.